### PR TITLE
Add a mode to generate a list of all matched items and their P31 values

### PR DESCRIPTION
**Pull request by lokal-profil**
This will hopefully allow us t oidentify a few false positives before
doing a live run.

Task: [T176950](https://phabricator.wikimedia.org/T176950)

### CODE REVIEW

**Comment by Vesihiisi** [comment]
For reference, current output looks like this: https://gist.github.com/Vesihiisi/ee8b37cae9ea4c1bbbee95bef5eee268

File: importer/importer.py [hunk 1]
```diff
@@ -209,11 +214,17 @@ def get_items(connection,
 
             uploader.upload()
             print("--------------------------------------------------")
+        if list_matches:
+            match_info = monument.print_matched_item_info_row()
+            if match_info:
+                utils.append_line_to_file(raw_data, matches_filename)
```
Comment (Vesihiisi) [inline importer/importer.py]: `match_info` instead of `raw_data`

File: importer/Monument.py [hunk 2]
```diff
@@ -49,8 +49,12 @@ def print_matched_item_info_row(self):
         """Generate a wikitext preview row for any matched item."""
         if not self.wd_item["wd-item"]:
             return None
-        row = "* {id} > {qid}: {{{{Data|item={qid}|property=p31}}}}".format(
-            id=self.monuments_all_id, qid=self.wd_item["wd-item"])
+        qid = self.wd_item["wd-item"]
+        qid_template = utils.wd_template("Q", qid)
+        data_template = "{{{{Data|item={qid}|property=p31}}}}".format(qid)
```
Comment (Vesihiisi) [inline importer/Monument.py]: `.format(qid=qid)`

File: importer/Monument.py [hunk 3]
```diff
@@ -49,8 +49,12 @@ def print_matched_item_info_row(self):
...
+        row = "* [wlm_url {id}] > {qid}: {data}".format(
```
Comment (Vesihiisi) [inline importer/Monument.py]: wrap `wlm_url` in curly braces

**Comment by Vesihiisi** [comment]
Updated output: https://gist.github.com/Vesihiisi/ee8b37cae9ea4c1bbbee95bef5eee268

**Comment by lokal-profil** [comment]
Output looks good to me and ends up like [:d:User:André Costa (WMSE)/test](https://www.wikidata.org/wiki/User:Andr%C3%A9_Costa_(WMSE)/test) on wiki

**Pull request merged**